cmd/fyne/internal/commands/build.go

```diff
@@ -197,14 +197,19 @@ func (b *Builder) build() error {
 		}
 	}
 
+	srcdir, err := b.computeSrcDir(fyneGoModRunner)
```

Comment:
Is there a reason this is not fixed earlier in the build validation? Not sure about the possible impact of using a difference srcdir for build compared to other phases...

Reply:
The reason is that once we call go build, it should build the binary where we expect too. The problem fixed here is to locate where the Icon and the metadata should be as that depend on where go build is going to look for the source code.